.github/workflows/build-llvm-image.yml

```diff
@@ -4,67 +4,68 @@
 name: Build x86 LLVM-Nightly Image
```

Comment:
The name here (and the name of the file) don't really match this "build & test" workflow